src/prefect/task_worker.py

```diff
@@ -350,7 +365,7 @@ async def execute_task_run(self, task_run: TaskRun):
     async def __aenter__(self):
         logger.debug("Starting task worker...")
 
-        if self._client._closed:
+        if self._client._closed:  # pyright: ignore[reportPrivateUsage]
```

Comment:
Perhaps the client should have a `closed` property?

Reply:
yes i think it probably should - I think there's a lot of places to update this so I can follow up in a later PR